docs/guide/contributing.rst

```diff
@@ -1,73 +1,206 @@
+* Documentation best practices.
+* Developer tools or docs-as-code toolchains and workflows.
+* High-level discussions of common tools. 
+
+For help finding where a new set of topics best fit into the rest of 
+the guide, ask in Slack or file a GitHub issue.
+
+Guidelines
+-----------
+
+The guide is not a place to plug your favorite toolchain, even if it's open source. 
+Follow these guidelines for discussing tools: 
+
+* Ensure your content is of interest to a general audience.
+* Avoid advocacy.
+* Present specific use cases, how problems were solved, and what worked or didn't work well. 
+* Consider first creating a GitHub issue or contacting one of the guide editors.
+
+Keep content focused on general principles and best practices instead of arguing over minor 
+points at the expense of major principles of clarity and communication. Avoid preferential 
+editing that only offers your personal preferences for dealing with a particular writing 
+situation. For example, if you recommend a particular word choice, tell the audience why it 
+matters.
+
+Use a friendly and encouraging tone. It's a good way to write docs in general, so practice 
+with docs about docs!
+
+When linking to resources, attribute the links appropriately and mention why the resources 
+are useful.
+
+Some files in the guide are written in `Markdown </guide/writing/markdown/>`_, and some 
+are written in `reStructured Text (rST) </guide/writing/reStructuredText/>`_. We prefer rST 
+although either format is acceptable.
+
+Editing the Write the Docs website
+-----------------------------------
+
+If you've never worked with Git and GitHub, use this section to get started. 
+
+To suggest small changes, `submit issues <https://github.com/writethedocs/www/issues>`_ via 
+the GitHub interface. 
+
+For changes that span multiple files, create a fork and branch, work locally (although you can 
+do most things in the GitHub interface in your branch), and keep your fork in sync with the main 
+repository.
+
+We encourage you to learn some of the basic tools we use but, if you don't want to deal with 
+GitHub, email attachments or inline text to guide@writethedocs.org.
+
+Prerequisites
+~~~~~~~~~~~~~~
+
+1. `Create a GitHub account`_.
+2. `Download and install Git`_.
+3. Open a terminal window and follow the instructions to `associate your
+   GitHub username with your local Git installation`_.
+
+   1. In macOS: Open the **Terminal** app.
+   2. In Windows: From the Start Menu, open **Git Bash**.
+
+4. In the repo UI, find the file name corresponding to an `existing issue`_ or page you want to improve. 
+
+For example, ``/docs/documentarians.rst`` creates https://www.writethedocs.org/documentarians/. 
+
+5. Review formatting guidelines for the Markup style the file uses. The example file above uses `reStructuredText (.rst) Markup`_. Other Markup styles include Markdown (.md) and AsciiDoc (.adoc).
+
+Update a guide in git
+~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This reads off to me, but I understand where we're coming from.

Reply:
I'll take a look at redoing it.

Reply:
To make matters worse, we have the headings "Contributing to the Write the Docs guide," "Editing the Write the Docs website," "Update a guide..."  I'll fix that too.

Reply:
I cleaned up the organization of the doc and renamed that section for clarity.